lib/router/router.js

```diff
@@ -109,6 +126,15 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && this.options.ifNoOperation && this.options.ifNoOperation.justForward === true) {
```

Comment:
I tested the useOriginal and it seems it just avoid the creating of the optimized original. I disable the optimized original using the following router options -> router: { originalSteps: false }
so noop query would make senses, but how could we handle the content-type?